lib/telegram/bot/api/endpoints.rb

```diff
@@ -0,0 +1,103 @@
+        'sendDocument' => Types::Message,
+        'sendVideo' => Types::Message,
+        'sendVoice' => Types::Message,
+        'sendVideoNote' => Types::Message,
+        'sendMediaGroup' => Types::Array.of(Types::Message),
+        'sendLocation' => Types::Message,
+        'editMessageLiveLocation' => Types::Message | Types::Bool,
+        'stopMessageLiveLocation' => Types::Message | Types::Bool,
+        'sendVenue' => Types::Message,
+        'sendContact' => Types::Message,
+        'sendChatAction' => Types::Bool,
+        'getUserProfilePhotos' => Types::UserProfilePhotos,
+        'getFile' => Types::File,
+        'banChatMember' => Types::Bool,
+        'unbanChatMember' => Types::Bool,
+        'restrictChatMember' => Types::Bool,
+        'promoteChatMember' => Types::Bool,
+        'leaveChat' => Types::Bool,
+        'getChat' => Types::Chat,
+        'getChatAdministrators' => Types::Array.of(Types::ChatMember),
+        'exportChatInviteLink' => Types::String,
+        'setChatPhoto' => Types::Bool,
+        'deleteChatPhoto' => Types::Bool,
+        'setChatTitle' => Types::Bool,
+        'setChatDescription' => Types::Bool,
+        'pinChatMessage' => Types::Bool,
+        'unpinChatMessage' => Types::Bool,
+        'getChatMembersCount' => Types::Integer,
+        'getChatMember' => Types::ChatMember,
+        'setChatStickerSet' => Types::Bool,
+        'deleteChatStickerSet' => Types::Bool,
+        'answerCallbackQuery' => Types::Bool,
+        'editMessageText' => Types::Message | Types::Bool,
+        'editMessageCaption' => Types::Message | Types::Bool,
+        'editMessageReplyMarkup' => Types::Message | Types::Bool,
+        'deleteMessage' => Types::Bool,
+        'sendSticker' => Types::Message,
+        'getStickerSet' => Types::StickerSet,
+        'uploadStickerFile' => Types::File,
+        'createNewStickerSet' => Types::Bool,
+        'addStickerToSet' => Types::Bool,
+        'setStickerPositionInSet' => Types::Bool,
+        'deleteStickerFromSet' => Types::Bool,
+        'answerInlineQuery' => Types::Bool,
+        'sendInvoice' => Types::Message,
+        'answerShippingQuery' => Types::Bool,
+        'answerPreCheckoutQuery' => Types::Bool,
+        'sendGame' => Types::Message,
+        'setGameScore' => Types::Message | Types::Bool,
+        'getGameHighScores' => Types::Array.of(Types::GameHighScore),
+        'setPassportDataErrors' => Types::Bool,
+        'editMessageMedia' => Types::Message | Types::Bool,
+        'sendAnimation' => Types::Message,
+        'sendPoll' => Types::Message,
+        'stopPoll' => Types::Poll,
+        'setChatPermissions' => Types::Bool,
+        'setChatAdministratorCustomTitle' => Types::Bool,
+        'sendDice' => Types::Message,
+        'getMyCommands' => Types::Array.of(Types::BotCommand),
+        'setMyCommands' => Types::Bool,
+        'deleteMyCommands' => Types::Bool,
+        'setStickerSetThumbnail' => Types::Bool,
```

Comment:
There was `setStickerSetThum`, but I found info about renaming in changelog:

![image](https://github.com/atipugin/telegram-bot-ruby/assets/6510020/89731062-2fb9-4c44-8cc4-6c835f280f3e)

But we also should rename all `thumb` fields.

Reply:
I've created #286 for fields.

Reply:
Now this PR covers it.